### DISCUSSION TITLE
Best way to install?

### DISCUSSION THREAD

**Discussion by gitihobo** [voicepaw/so-vits-svc-fork#777] [General]
I want to install anaconda and git clone this repo, what do you do to install it after that?

**Comment by AKhilRaghav0** [comment]
just Activate Env and follow the same steps
```
svc pre-resample
svc pre-config
svc pre-hubert
svc train -t
```

place all datasets in dataset_raw directory of so-vits-svc-fork
it should be like that
"`so-vits-svc-fork/dataset_raw/singer_name/*tracks.wav`"
